### PR TITLE
Windows enable payloads > 2560

### PR DESCRIPTION
**Pull request by TheOneRing**
Depends on https://github.com/frankosterfeld/qtkeychain/pull/250

### CODE REVIEW

**Comment by TheOneRing** [comment]
ping :D

File: qtkeychain/keychain_win.cpp [hunk 1]
```diff
@@ -11,14 +11,23 @@
...
 #include <wincrypt.h>
 
 #include <memory>
 
 using namespace QKeychain;
 
 namespace {
-    QString formatWinError(unsigned long errorCode)
+    constexpr std::wstring_view PRODUCT_NAME = L"QtKeychain";
+    constexpr std::wstring_view ENCRYPED_DATA_KEY = L"QtKeychain-encrypted data";
```
Comment (frankosterfeld) [inline qtkeychain/keychain_win.cpp:23]: typo ENCRYPED => ENCRYPTED

File: qtkeychain/keychain_win.cpp [hunk 2]
```diff
@@ -9,18 +9,91 @@
...
 #include <windows.h>
+#include <wincred.h>
 #include <wincrypt.h>
 
 #include <memory>
 
 using namespace QKeychain;
 
+namespace {
+    constexpr std::wstring_view PRODUCT_NAME = L"QtKeychain";
```
Comment (frankosterfeld) [inline qtkeychain/keychain_win.cpp:22]: Hmm, this would require C++17, can we do without views?

Comment (TheOneRing) [reply qtkeychain/keychain_win.cpp:22]: sure

**Comment by frankosterfeld** [comment]
Thanks!

**Pull request merged**